integration-cli/docker_cli_build_test.go

```diff
@@ -3695,6 +3695,49 @@ func TestBuildDockerignore(t *testing.T) {
 	logDone("build - test .dockerignore")
 }
 
+func TestBuildDockerignoreExclude(t *testing.T) {
+	name := "testbuilddockerignoreexclude"
+	defer deleteImages(name)
+	dockerfile := `
+        FROM busybox
+        ADD . /bla
+               RUN [[ -f /bla/src/x.go ]]
+               RUN [[ -f /bla/Makefile ]]
+               RUN [[ ! -e /bla/src/_vendor ]]
+               RUN [[ ! -e /bla/.gitignore ]]
+               RUN [[ ! -e /bla/README.md ]]
+               RUN [[ ! -e /bla/.git ]]
+               RUN [[ -f /bla/LICENSE.md ]]
+               RUN [[ -f /bla/docs/config.json]]`
```

Comment:
let's add another (non-md) file under "docs/" and make sure its not there.  I was going to suggest we test for "useless.md" but that's excluded by the *.md.  I want to make sure the "docs" line catches everything except config.json.